docs/sql-guide/functions/function-totimestamp.md

```diff
@@ -0,0 +1,78 @@
+---
+title: TOTIMESTAMP()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+
+# TOTIMESTAMP() function
+
+The `TOTIMESTAMP()` function converts the given integer value to a timestamp and returns it. It uses the [unix epoch](https://www.unixtutorial.org/unix-epoch/) as the base and calculates date-time equivalent of the integer.
+
+## Syntax
+
+```
+TOTIMESTAMP(int_exp, timeunit)
+```
+
+## Arguments
+
+
+| Argument | Description | Data type |
+|---|---|---|
+| `int_exp` | Required `int` value or expression to be converted to a `timestamp`. | `int` |
+| `timeunit` | Optional `string` value specifying the granularity of the input `int_exp`, defaults to `s`. see [TIMEUNIT values](#timeunit-value).| `string` |
+
+
+## Returns
+
+| Data type | Value |
+|---|---|
+| `timestamp` | Returns timestamp equivalent of `int_exp`, it converts the input using unix epoch as base and the optional `timeunit` to determine the granularity of the input value. |
+
+## Remarks
+If `timeunit` is not supplied, the granularity of the input will default to seconds.
+
+### TIMEUNIT value
+
+| Unit | Declaration |
+|---|---|
+| seconds (default) | `s` |
+| milliseconds | `ms` |
+| microseconds | `us` |
+| nanoseconds | `ns` |
+
+## Examples
+
+### Integer value as input to timestamp column. 
+Literal integers supplied as input to timestamp columns are implicitly converted to timestamp, implicit conversion of integer to timestamp will treat the integer value as seconds since unix epoch. For integer date-time values defined at granularity other than seconds TOTIMESTAMP() can be used to convert them to timestamp by selecting appropriate `timeunit` parameter that matches the granularity of the input. 
```

Comment:
Can we summarise this information? For example:

```
### Convert integer value to seconds
```
This would mean the first sentence can be removed.

The second sentence is repeating what's already been explained in the rest of the file, so can also be removed.

Reply:
Alternatively, if you **really** want to make the point in the second sentence, write an example for the user.